### DISCUSSION TITLE
pnpm 问题

### DISCUSSION THREAD

**Discussion by ABC-xiaoxuan** [YunYouJun/valaxy#133] [Q&A]
![image](https://user-images.githubusercontent.com/72299627/205665851-de92844c-422f-4c0e-bf10-91957dbc953b.png)
在使用pnpm创建时，我按照命令行提示，输入y，随后让我选择npm还是pnpm我选择了pnpm遇到了这个问题，请问应该怎么解决

**Comment by YunYouJun** [comment]
已经成功创建了，没有问题。
这个是警告，不是错误。

**Comment by YunYouJun** [comment]
你的 pnpm 版本是多少？

https://github.com/pnpm/pnpm/issues/4183
